citrination_client/models/client.py

```diff
@@ -129,9 +137,54 @@ def _get_predict_body(self, candidates, method="scalar", use_prior=True):
             }
         }
 
+    def submit_predict_request(self, data_view_id, candidates, prediction_source='scalar', use_prior=True):
+        """
+        Submits an async prediction request.
+
+        :param data_view_id: The id returned from create
+        :param candidates: Array of candidates
+        :param prediction_source: 'scalar' or 'from_distribution'
```

Comment:
`scalar_from_distribution`?

Reply:
Ah, thank you! Good catch. I fixed it in `predict` but not here.